web/src/constants/dimensions.ts

```diff
@@ -19,7 +19,7 @@ const dimensions: DimensionsType = {
   minMaxWidth: 'screen and (min-width: 1100px)',
   maxWidth: 1100,
   toolbarWidth: 125,
-  toolbarHeight: 73,
+  toolbarHeight: 44,
```

Comment:
Why did you change this?

Reply:
because i decreased the toolbar height (some icon padding). it was not necessary to have such a big height